src/tools/fuzzing.h

```diff
@@ -2652,6 +2660,20 @@ class TranslateToFuzzReader {
     return builder.makeRefEq(left, right);
   }
 
+  Expression* makeI31New(Type type) {
+    assert(type == Type::i31ref);
+    assert(wasm.features.hasReferenceTypes() && wasm.features.hasGC());
+    auto* value = make(Type::i32);
+    return builder.makeI31New(value);
+  }
+
+  Expression* makeI31Get(Type type) {
+    assert(type == Type::i32);
+    assert(wasm.features.hasReferenceTypes() && wasm.features.hasGC());
+    auto* i31 = make(Type::i31ref);
+    return builder.makeI31Get(i31, oneIn(2));
```

Comment:
This might make it clearer that this is choosing a random bool.

```suggestion
    return builder.makeI31Get(i31, bool(oneIn(2)));
```